### PR TITLE
Portfolio-mobile-first

### PR DESCRIPTION
**Pull request by cosywasswa**
- Created a contact form with all elements
-  Styled form elements with CSS
-  Updated the README file with more form features
-  Corrected linter errors

### CODE REVIEW

File: index.css [hunk 1]
```diff
@@ -1,619 +1,496 @@
...
+    position: relative;
+    top: 25px;
+    left: 90px;
+    background-color: #ff6b00;
+    border: 2px solid #ff6b00;
+  }
+
+  form button:hover {
+    border-color: green;
+  }
```
Comment (usorfaitheloho) [inline index.css]: Great job implementing most of the design in the form, please add the original color for the button interaction in your design, the `hover` styling color is not green as you have it here. it's really important to be consistent with the design given to give a better user interface and experience to visitors 😄 

Button interaction styling :

image

Your design :

image

File: index.html [hunk 2]
```diff
@@ -245,6 +245,33 @@ <h4>Skills</h4>
...
+        </div>
+        <form action="https://formspree.io/f/mjvdlwyk" method="post">
+          <fieldset class="field">
+            <input type="text" name="Full-name" id="full-name" placeholder="Full name" maxlength="30" required />
+            <input type="email" name="email-address" id="email" placeholder="Email address" required /> 
+          </fieldset>
+          <textarea name="comments" maxlength="500" placeholder="Write me something..." required></textarea>
+          <button type="submit"><h5>Get in touch</h5></button>
+        </form>
+        <hr>
```
Comment (usorfaitheloho) [inline index.html:262]: Please kindly follow the design for the `hr` tag, so as to have a good UI for your form interaction 😄 

Figma design :

image

Your design :

image

**Review by usorfaitheloho — CHANGES_REQUESTED** [review]
> Hi @cosywasswa  ,
> 
> Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!  
> 
> ## Highlights
> * Followed Figma design ✔️
> * Descriptive pull request ✔️
> * Used GitHub flow ✔️
> 
> ## Required Changes ♻️
> _Check the comments under the review._
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.css [hunk 3]
```diff
@@ -1,619 +1,511 @@
...
+    color: #fff;
+  }
+
+  .form-area hr {
+    width: 375px;
+    height: 1px;
+    position: relative;
+    top: 125px;
+    left: -28px;
+  }
```
Comment (franclobo) [inline index.css:490]: - **[OPTIONAL]** Please 🙏🏽 use the color from the Figma design `#DFE1E6` to match with the original design.

**Review by franclobo — APPROVED** [review]
Hi @cosywasswa and @Rifat8080, 

# STATUS: APPROVED  😃 

git-merge

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- No linter errors ✔️ 
- Correct Github flow ✔️ 
- Descriptive PR ✔️ 
- Good UI ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**